### PR TITLE
[RORO-4789][JT] Guidance related to Stop 24 and DWD on Inspection req…

### PR DESCRIPTION
**Pull request by tianjianfeng**
…uired page

### CODE REVIEW

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html [hunk 1]
```diff
@@ -81,6 +82,21 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_export.arriving.h3")</h
         <li><a href="@{appConfig.checkDelaysInlandBorderFacilityUrl}">@messages("inspection_needed_export.arriving.p1.link2")</a></li>
     </ul>
 
+    @govukInsetText(InsetText(
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html:85]: is this change needed only for export but not for import ?

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html [hunk 2]
```diff
@@ -81,6 +82,21 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_export.arriving.h3")</h
...
+
+    <p class="govuk-body">@messages("inspection_needed_export.arriving.p2")</p>
+    <ul class="govuk-list govuk-list--bullet" >
+        <li>@messages("inspection_needed_export.arriving.li.1")</li>
+        <li>@messages("inspection_needed_export.arriving.li.2")</li>
+        <li>@messages("inspection_needed_export.arriving.li.3")</li>
+    </ul>
+
+    <p class="govuk-body">
+        @messages("inspection_needed_export.arriving.p3") <a href="https://www.gov.uk/government/publications/attending-an-inland-border-facility/attending-an-inland-border-facility#if-your-vehicle-is-excluded">
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html:97]: minor: there is a hmrcNewTabLink component that we can use here
you can see it here: https://github.com/hmrc/goods-movement-system-frontend/pull/1017/files#diff-47ba94c3b4a56fad3f538aa6fc029f94324eecd58585a121a7a190e21a2f08a4R25

**Review by sureshhmrc — DISMISSED** [review]

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html (hunk 1 above)
Comment (tipperba) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html:85]: I was about to say the same: the ticket says EU to GB as well.

Comment (sureshhmrc) [inline app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html:85]: are you not checking for direction check here ? if so it will show the content for all 4 directions?

Comment (tipperba) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html:85]: this particular view is just for GB to EU, I think

Comment (tianjianfeng) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html]: Don the import as well.

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html (hunk 2 above)
Comment (tianjianfeng) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html]: new tab link doesn't work. So still stick to the original.

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html (hunk 1 above)
Comment (tianjianfeng) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html]: Done

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html [hunk 3]
```diff
@@ -81,6 +83,20 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_import.arriving.h3")</h
         <li><a href="@{appConfig.attendingInlandBorderFacilityUrl}">@messages("inspection_needed_import.arriving.p1.link1")</a></li>
         <li><a href="@{appConfig.checkDelaysInlandBorderFacilityUrl}">@messages("inspection_needed_import.arriving.p1.link2")</a></li>
     </ul>
+    @govukInsetText(InsetText(
+        content = Text(messages("inspection_needed_export.arriving.inset"))
+    ))
```
[the remark this answers is not in the document]
Comment (tianjianfeng) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html]: Done the direction check for the UK_INBOUND

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html [hunk 4]
```diff
@@ -78,10 +78,20 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_import.arriving.h3")</h
...
+        <li><a href="@{
+            appConfig.attendingInlandBorderFacilityUrl
+        }">@messages("inspection_needed_import.arriving.p1.link1")</a></li>
+        <li><a href="@{
+            appConfig.checkDelaysInlandBorderFacilityUrl
+        }">@messages("inspection_needed_import.arriving.p1.link2")</a></li>
     </ul>
 
+
+    @if(isInbound) {
```
Comment (tipperba) [inline app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html:90]: I think there's potential for confusion with the boolean `isInbound`. Instead of having this boolean, can we just pass the direction in and check that it's EU to GB (which is admittedly poorly named UK_INBOUND)?

ie. just have a check that `direction == UK_INBOUND` within the view.

Comment (tianjianfeng) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html]: updated

File: app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html [hunk 5]
```diff
@@ -87,7 +88,7 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_import.arriving.h3")</h
     </ul>
 
 
-    @if(isInbound) {
+    @if(direction.equals(UK_INBOUND)) {
```
Comment (tipperba) [inline app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html:91]: Can we use `==` (or `===`) here instead? It's more idiomatic Scala and handles nulls better (not that null is a problem in this case)

Comment (tianjianfeng) [reply app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_import.scala.html]: updated.

**Review by tipperba — APPROVED** [review]
